aten/src/ATen/TensorIterator.cpp

```diff
@@ -570,7 +570,7 @@ bool TensorIteratorBase::is_dim_reduced(int dim) const {
 }
 
 void TensorIteratorBase::permute_dimensions(IntArrayRef perm) {
-  TORCH_INTERNAL_ASSERT(perm.size() == ndim());
+  TORCH_INTERNAL_ASSERT(perm.size() == (unsigned)ndim());
```

Comment:
I'm not familiar with this syntax; is this a cast of some sort?

Reply:
Good point, let me replace it with static_cast